src/main.js

```diff
@@ -385,7 +396,7 @@ export async function main() {
     .command(
       'create <basedir> <addr> <port>',
       'create a new Vat in BASEDIR',
-      yargs => {
+      _yargs => {
```

Comment:
This might be behavior-changing.. the argument passed into this arrow function might be different than the`yargs` object on which we invoked `.command`. I *think* I was just copy-pasting the yargs examples here, which probably did the shadowing thing (their recommended style might clash with eslint's no-shadow rule), but we should confirm that the two objects are EQ before changing this.

Reply:
Since this is what they have in the docs, I put a comment and reverted this